repose-aggregator/src/docs/asciidoc/recipes/valve-installation.adoc

```diff
@@ -0,0 +1,63 @@
+= Valve Installation
+:nexus-releases: https://maven.research.rackspacecloud.com/content/repositories/releases/org/openrepose
+
+include::../_includes/in-progress.adoc[]
+
+Covers the typical Repose setup -- a Valve installation (i.e., standalone application) running as a Linux service.
+
+For instructions on starting Repose after setup, see: <<running-valve.adoc#,Running Valve>>.
+
+For more information about the packages themselves, see: <<../architecture/packages.adoc#,Packages>>.
+
+== APT Package Manager Based Linux Distributions
```

Comment:
Drop this a level lower under an "Automated" section to match the "Manual" section.